src/auth/src/errors.rs

```diff
@@ -88,3 +89,34 @@ impl Display for CredentialError {
 pub enum InnerAuthError {
     // TODO(#389) - define error types here
 }
+
+#[allow(dead_code)] // TODO(#442) - implementation in progress
+pub(crate) fn is_retryable(c: StatusCode) -> bool {
+    c == StatusCode::INTERNAL_SERVER_ERROR
+        || c == StatusCode::SERVICE_UNAVAILABLE
+        || c == StatusCode::REQUEST_TIMEOUT
+        || c == StatusCode::TOO_MANY_REQUESTS
+}
```

Comment:
For clarity consider:

```suggestion
pub(crate) fn is_retryable(c: StatusCode) -> bool {
   match c {
       StatusCode::INTERNAL_SERVER_ERROR | StatusCode::SERVICE_UNAVAILABLE | StatusCode::REQUEST_TIMEOUT | StatusCode::TOO_MANY_REQUESTS => true,
        _ => false,
      }
}
```

Reply:
Done, plus a comment about `500`